packages/apps/plugins/plugin-deck/src/components/DeckLayout.tsx

```diff
@@ -197,31 +197,7 @@ const NodePlankHeading = ({
           )
         }
         close={part[0] === 'complementary' ? 'minify-end' : true}
-      >
-        {/* TODO(thure): This, and all other hardcoded `comments` references, needs to be refactored. */}
-        {node && !!node.data?.comments && !slug?.endsWith('comments') && (
```

Comment:
Were you able to verify that this button doesn't serve a purpose anymore (i.e. there's another button to open the complementary sidebar & it's content changes based on attention)?